default.qdbd.cfg

```diff
@@ -4,6 +4,9 @@
             "async_ts": {
                 "flush_deadline": 500
             }
+        },
+        "network": {
```

Comment:
Given that `start-services.sh` already does:
```
ARGS_INSECURE="--id ${NODE_ID} -a ${THIS_URI_INSECURE} -r ${THIS_DATA_DIR_INSECURE} -l ${THIS_LOG_DIR_INSECURE} --enable-performance-profiling --with-firehose \$qdb.firehose"
```
It will not change anything in the current behaviour.

But please remove `--enable-performance-profiling` from the script.
You could actually put firehose too into the default config.

Reply:
Ah yes but why I didn't get this version when getting submodules in .NET API ? I will put those options into the config.